README.md

```diff
@@ -110,20 +110,43 @@ poetry run make protos
 poetry run hathor-cli run_node --status 8080
 ```
 
+It may take a considerable amount of time for it to sync all the transactions in the network. To speed things up, read below.
+
+#### Speeding up the sync
+You can use database snapshots to speed things up.
+
+We provide both testnet and mainnet snapshots:
+- Testnet: https://hathor-public-files.s3.amazonaws.com/temp/testnet-data-20210707174940.tar.gz
+- Mainnet: https://hathor-public-files.s3.amazonaws.com/temp/data-20210624040233.tar.gz
```

Comment:
Check if it's possible to use a permanent link that just points to the most recent snapshot

Reply:
A proposed solution was commited